code/modules/reagents/reagents/medicine.dm

```diff
@@ -385,18 +385,19 @@
 /datum/reagent/medicine/peridaxon/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
 	if(ishuman(M))
 		var/mob/living/carbon/human/H = M
-		for(var/obj/item/organ/I in H.internal_organs)
+		var/list/organs_sans_brain_and_bones = H.internal_organs - H.internal_organs_by_efficiency[BP_BRAIN] - H.internal_organs_by_efficiency[OP_BONE] // Peridaxon shouldn't heal brain or bones
+		for(var/obj/item/organ/I in organs_sans_brain_and_bones)
 			var/list/current_wounds = I.GetComponents(/datum/component/internal_wound)
-			if(LAZYLEN(current_wounds) && !BP_IS_ROBOTIC(I)) //Peridaxon heals only non-robotic organs
+			if(LAZYLEN(current_wounds) && !BP_IS_ROBOTIC(I) && prob(25)) //Peridaxon heals only non-robotic organs
```

Comment:
```suggestion
			if(LAZYLEN(current_wounds) && !BP_IS_ROBOTIC(I) && prob(75)) //Peridaxon heals only non-robotic organs
```
I'll do this.

Reply:
Prob is there so it doesn't treat all wounds immediately.

Reply:
could also make it just pick one at random instead of fixing all.